dist/src/main/java/io/camunda/zeebe/gateway/StandaloneGateway.java

```diff
@@ -33,18 +31,39 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.elasticsearch.ElasticsearchRestClientAutoConfiguration;
+import org.springframework.context.ApplicationListener;
 import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.event.ContextClosedEvent;
 
-public class StandaloneGateway {
+@SpringBootApplication(exclude = {ElasticsearchRestClientAutoConfiguration.class})
```

Comment:
Because currently the gateway and the broker are part of the same final artifact, which includes the Elasticsearch Exporter. I think we may not have been specifying the packages to scan before, so it would have gotten picked up via the ES library. It's possible this isn't necessary since we are explicit about package scanning, I'd have to try it to see.

Reply:
It looks like it will still find it and instantiate an ES client (including a thread pool). I'm not an expert enough, so for this PR let's keep it, but we can look into it together next week.